src/app/shared/styles/workflow-container.component.scss

```diff
@@ -23,3 +23,9 @@ a.underline-none {
   width: 14rem;
   text-align: start;
 }
+
+:host ::ng-deep .mat-tab-label {
+  padding-left: 0.5em !important;
+  padding-right: 0.5em !important;
```

Comment:
Initially, I thought it might make sense to increase the total minimum padding to a little more than 1em (0.5em + 0.5em).  However, in the "super narrow screen" scenario, it's probably preferable to have the options jammed together, even though it's a tad ugly, rather than need to use a button to see them.